web/src/message_list_view.ts

```diff
@@ -504,15 +574,32 @@ export class MessageListView {
         };
     }
 
-    is_current_message_list() {
+    is_current_message_list(): boolean {
         return this.list === message_lists.current;
     }
 
     get_calculated_message_container_variables(
-        message,
-        existing_include_sender,
+        message: Message,
+        existing_include_sender: boolean,
         is_revealed = false,
-    ) {
+    ): {
+        timestr: string;
+        background_color?: string;
+        small_avatar_url: string;
+        sender_is_bot: boolean;
+        sender_is_guest: boolean;
+        should_add_guest_indicator_for_sender: boolean;
+        is_hidden: boolean;
+        mention_classname?: string | null;
```

Comment:
Is this both nullable and possibly undefined?

Reply:
Yeah, based off of how the code was when I was working on. Ideally this wouldn't be the case, though. I didn't look much into what it would involve to change this though.